packages/arb-token-bridge-ui/src/components/common/Button.tsx

```diff
@@ -67,7 +67,6 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
         >
           {showLoader && (
             <Loader
-              type="TailSpin"
               color={loadingProps?.loaderColor || 'white'}
               width={loadingProps?.loaderWidth || 16}
```

Comment:
❓ Do we want to allow custom loader width? The prop doesn't seem to be used anywhere.